open-banking-accelerator/components/event-notifications/com.wso2.openbanking.accelerator.event.notifications.service/src/main/java/com/wso2/openbanking/accelerator/event/notifications/service/util/EventNotificationServiceUtil.java

```diff
@@ -151,4 +154,15 @@ public static String getCallbackURL(String clientID) {
     public static DefaultEventCreationServiceHandler getDefaultEventCreationServiceHandler() {
         return new DefaultEventCreationServiceHandler();
     }
+
+    /**
+     * Method to map Event subscription Service error to API response.
+     * @return EventNotificationErrorDTO
```

Comment:
Please complete method comment by adding @param details as well.

Reply:
Fixed in https://github.com/wso2/financial-open-banking/pull/3/commits/a31eef6176e5003bd359e183aae45a90a5874f2b